OpenIDPlugin.inc.php

```diff
@@ -150,6 +150,7 @@ public function callbackLoadHandler($hookName, $args)
 		$op = $args[1];
 		$request = Application::get()->getRequest();
 		$templateMgr = TemplateManager::getManager($request);
+		AppLocale::requireComponents(LOCALE_COMPONENT_PKP_COMMON, LOCALE_COMPONENT_APP_COMMON, LOCALE_COMPONENT_PKP_USER);
```

Comment:
Why are we adding those locale components? Do you see untranslated locale keys when using the plugin?

Reply:
@defstat Yes, some pages are not displaying the translations correctly.